### ISSUE TITLE
Feat: Adapt Schema used by EventBuilder for "fields" to real need

### ISSUE THREAD

**Issue by inkhey** [tracim/tracim#3058]
## Feature description

In event builder for live messages (see PR https://github.com/tracim/tracim/pull/3022), we do use these schema:
```
    _user_schema = UserDigestSchema()
    _workspace_schema = WorkspaceDigestSchema()
    _content_schema = ContentDigestSchema()
    _event_schema = EventSchema()
```
Some of these schema return too much information (like `sidebar_entries` for workspace), so we probably should limit the data returned.
To do this, we need to clarify with frontend what is really needed, what is nice to have and what is useless to put there.

**Comment by grignards** [comment]
I used already existing schemas as returned by the classic HTTP APIs in order to limit the number of representations we have for an entity. If some information are not needed, we should remove them from those schemas instead of creating new ones (IMHO).

**Comment by inkhey** [comment]
> I used already existing schemas as returned by the classic HTTP APIs in order to limit the number of representations we have for an entity. If some information are not needed, we should remove them from those schemas instead of creating new ones (IMHO).

I suggest creating new one and renaming existing one (digest schema are not really digest in some case), and discuss with frontend about usage, because as far i remembers, some of these digest schema where digest… until we do need more information for some specific usage.

**Comment by grignards** [comment]
After discussing with @Skylsmoi we decided to include the same data as in the HTTP apis for the corresponding entity (only exception: the contents of html documents).

This means using the xxxSchema for serializing.

**Comment by Skylsmoi** [comment]
As @gbonaspetti and @metzgegu are working on the implementation of TLM for app HtmlDoc and app File, some issues pop up:
- The TLM content.modified includes a `content` body.
  - it lacks the revision_type which we need to update the timeline. One solution would be to return an object `revision` instead of `content`. It would be easier for front, but @inkhey seems to point out the fact that it might add inconsistency
- Since the `content.modified` TLM doesn't return the raw_content property, we need a way to get it.
  - One solution would be to have a specific TLM for raw_content changes. This would be ideal for htmldoc, but a bit odd for file since raw_content is used for the description

Side note: for frontend, having TLM content.modified returning different objects in terms of content_type isn't an issue.

**Comment by inkhey** [comment]
Partially fixed by #3086 (merged in order to get unbroken tests in backend).
